qst/src/main/java/io/deephaven/qst/table/TimeTable.java

```diff
@@ -85,6 +91,8 @@ public interface Builder {
 
         Builder startTime(Instant startTime);
 
+        Builder blinkTable(boolean blinkTable);
+
         Builder id(UUID id);
```

Comment:
I think you might need `of` overloads for `blinkTable`, or to randomize `id` by default. Otherwise, I worry users will mess up `id` when using the builder.